plugins/quantum_k_means/__init__.py

```diff
@@ -20,7 +20,7 @@
 from qhana_plugin_runner.util.plugins import plugin_identifier, QHAnaPluginBase
 
 _plugin_name = "quantum-k-means"
-__version__ = "v1.2.0"
+__version__ = "v0.2.0"
```

Comment:
skipping to a major version was ok, but here there was a major+minor version change, hence the comment about skipping multiple versions. But it is ok to leave it like it is now.